exporter/opencensusexporter/opencensus.go

```diff
@@ -116,9 +116,12 @@ func (oce *ocagentExporter) ProcessTraceData(ctx context.Context, td data.TraceD
 		},
 	)
 	if err != nil {
+		// TODO: If failed to send all maybe record a different metric. Failed to "Sent", but
+		// this may not be accurate if we have retry outside of this exporter. Maybe the retry
+		// processor should record these metrics.
 		return err
 	}
-	nSpansCounter := internal.NewExportedSpansRecorder("ocagent")
-	nSpansCounter(ctx, td.Node, td.Spans)
+	ctxWithExporterName := internal.ContextWithTraceExporterName(ctx, "ocagent")
```

Comment:
the receiver is "opencensus" I think that it makes more sense to say the same here, it is not about ocagent it is about transport/serialization format.

Reply:
Rename it to oc_trace. See comments why we should include the signal in the name (e.g. exporting metrics from traces).